src/App.css

```diff
@@ -59,3 +82,165 @@ body {
     transform: rotate(360deg);
   }
 }
+
+.action {
+  width: 4.063rem;
+  height: 1.125rem;
+  margin: 0 1.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  padding: 0 0 0 1.2%;
+}
+
+.title {
+  width: 20.313rem;
+  height: 1.813rem;
+  margin: 0.488rem 20.188rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+  padding: 0 0 0 1.2%;
+}
+
+.author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  padding: 0 0 0 1.2%;
+}
+
+.comment {
+  width: 4.563rem;
+  height: 1.198rem;
+  margin: 1.188rem 0.938rem 0.125rem 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.999rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: #fff;
+  border: none;
+}
+
+.span {
+  width: 0.1rem;
+  height: 1.5rem;
+  margin: 1rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+  background-color: #e8e8e8;
+}
+
+.progress-bar-c {
+  display: flex;
+  gap: 1.25rem;
+  width: 30%;
+}
+
+.progress-bar {
+  height: 70px;
+  width: 70px;
+}
+
+.progress-value-c {
+  display: flex;
+  flex-direction: column;
+  margin-top: 5px;
+}
+
+.progress-percentage {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 30px;
+  margin: 0;
+  color: #121212;
+  font-weight: 500;
+}
+
+.progress-completed {
+  font-family: 'Montserrat', sans-serif;
+  margin: 5px 0 0 0;
+  color: #121212;
+  opacity: 0.5;
+  font-size: 15px;
+}
+
+.update-progg-c {
+  display: flex;
+  flex-direction: column;
+}
+
+.chapter-h2 {
+  width: 11.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter-h3 {
+  width: 7.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
- [ ] I noticed that some of these properties like the ones I mention below are repeated severally in your CSS. I suggest you create a class for these properties and use the class on the element that needs the class instead of repeating them in the CSS for almost every class. This is to achieve [CSS best practices](https://speckyboy.com/good-bad-css-practices/) 🤔.
 ```CSS
  font-family: 'RobotoSlab', sans-serif;
  font-weight: 300;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;